src/test/resources/database-migrate-data.sql

```diff
@@ -0,0 +1,80 @@
+-- Harvested from a mod-calendar 1.12.0 using the contents of AbstractExistingCalendarTest
+INSERT INTO
+  "test_mod_calendar"."openings" ("id", "jsonb", "creation_date", "created_by")
+VALUES
+  (
+    'cccccccc-cccc-cccc-cccc-cccccccccccc',
```

Comment:
Personally, I would use valid UUIDs for these values.

Some modules check whether a UUID is in the expected form and that might (it may already have) become a standard expectation for modules in the future.

I also think it is good for sample / test data to reflect real expectations where possible.

Reply:
These UUIDs are valid, I just changed them from randomly-generated ones to ones that were a bit easier to read/follow throughout the sql and tests

Reply:
bq. These UUIDs are valid

I believe FOLIO conventionally moved to use a specific sub-set of the available UUID schemes, for example, https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/uuid.json

Though I don't think is very well documented and likely many folks don't know about it.

Reply:
Ah, looks like the main restriction is just that the version/variant must not be zero/otherwise invalid.  I can update the test values to have valid versions/variants (e.g. `cccccccc-cccc-4ccc-8ccc-cccccccccccc`), if that would be preferred.

Reply:
Done